packaging/kubernetes-deployment/sample-ebpfprograms/gocounter-bytecode.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: bpfd.io/v1alpha1
```

Comment:
@astoycos Is there a way to specific which node to load program on? What if we want a different interface for each node (like load on the nginx interface, which would be different on each node)?

Reply:
Not Yet, but I'm hoping to add that functionality ASAP